test/initial.test.js

```diff
@@ -1,13 +1,34 @@
 import chai from "chai";
+import jwt from "jwt-simple";
```

Comment:
Can we just stick to `jsonwebtoken` package
```suggestion

```

Reply:
jwt-simple package helps us to use metods like encode and decode jwt but jsonwebtoken allows only sign and verify to be used.

Reply:
can't we use both of them?